parm/config/gfs/config.ufs

```diff
@@ -299,8 +299,8 @@ case "${fv3_res}" in
         export DELTIM=150
         export layout_x_gdas=8
         export layout_y_gdas=16
-        export layout_x_gfs=8
-        export layout_y_gfs=16
+        export layout_x_gfs=16
+        export layout_y_gfs=24
```

Comment:
@AntonMFernando-NOAA FYI in reference to #3420 the GFS portion of C1152 will be changing to this layout.